merlin/config/utils.py

```diff
@@ -8,6 +8,11 @@ class Priority(enum.Enum):
     mid = 2
     low = 3
 
+def is_rabbit_broker(broker):
```

Comment:
Was PyLint complaining about complexity or line length for having these 1 line `in` examinations in the body of get_priority()? Unless we anticipate maybe growing the supported brokers (do we?), this seems a simple enough check we could embed it in get_priority() without a function lookup, but it's just stylistic difference, looks good otherwise.

Reply:
this is mostly to future-proof against possible other brokers in the future. we have no plans, but celery supports a bunch of different ones.

Reply:
also a bunch of different kinds of brokers can be classified as either rabbit or redis brokers (eg depending on ssl), so it's nice to be able to add them here